onnxruntime/build.rs

```diff
@@ -0,0 +1,14 @@
+use std::env;
+
+fn main() {
+    if boolean_env_var("ORT_ALLOW_DEBUG_LOGGING") {
```

Comment:
今思い付きましたが、デバッグビルドでは有効化扱いしていいかもしれません。

```suggestion
    if cfg!(debug_assertions) || boolean_env_var("ORT_ALLOW_DEBUG_LOGGING") {
```

Reply:
初学者に優しくて良いのかなと思いました！

Reply:
デバッグビルドでは有効化されるようにしました。